Mathlib/Tactic/Linter/Common.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2024 Moritz Firsching All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa, Moritz Firsching
+-/
+
+import Lean.Elab.Command
+
+/-!
+This file contains functions that are used by some linters.
+-/
+open Lean Parser Elab Command Meta
+namespace Mathlib.Linter
+
+/--
+If `stx` is a declaration, then `getNames stx` returns the array of identifiers
+for the name of the declarations whose syntax range is contained in `stx`.
+
+If `stx` is not a declaration, then `getNames stx` returns `#[]`.
```

Comment:
```suggestion
```
Looking back at this, I am not so sure that this line is necessary: I think that a non-declaration that happens to add a declaration to the environment would actually produce a name.  Besides, if there are no names to be extracted, clearly the output is going to be `#[]`, so it is probably clearer to remove this last sentence.

Reply:
I removed the line